packages/utils/src/twitter.ts

```diff
@@ -0,0 +1,30 @@
+const BASE_TWITTER_URL = "https://twitter.com"
+
```

Comment:
Lets do an object like "twitterParser" that includes three functions

Because now they have more generic names, and we can add `twitter` prefix or put all functions to object